src/widgets/statusCard/StatusCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import * as S from './StatusCard.style';
+import { StatusTag } from '@/features/myPage';
+
+interface StatusCardProps {
+  title: string;
+  content: string;
+  status?: 'rejected' | 'accepted' | 'pending' | undefined;
+}
+
+const StatusCard = ({ title, content, status }: StatusCardProps) => {
+  const activateButton = status === 'pending' || status === undefined;
```

Comment:
이런 느낌으로 표현되면 어떨까 싶습니다~
현재 사용되는 방식은 이 조건식을 기준으로 렌더링이 되냐 안되냐가 갈리는데, 기존의 이름은 스타일 처리와 관련된 느낌의 변수처럼 느껴지네요!
```suggestion
  const showActivateButton = status === 'pending' || status === undefined;
```

Reply:
[05a862b](https://github.com/GrebbMe/grabbme-fe/pull/91/commits/05a862b3a8db3228f24de6b2ab14aba1316b44b4) 에 반영했습니다! 약간의 어색함을 느끼고 있었는데, 동사형으로 처리하면 훨씬 직관적이네요! 감사합니다 :)